src/middlewares/validate.js

```diff
@@ -0,0 +1,27 @@
+const { validate, Symbols } = require('helpers/validator');
+const _ = require('lodash');
+
+const validateMiddleware = (validators = []) => async (ctx, next) => {
+  const payload = {
+    ...ctx.request.body,
+    ...ctx.query,
+    ...ctx.params,
+    [Symbols.PERSISTENT]: _.pick(ctx, ['request.body', 'query', 'params', 'state']),
+  };
+
+  const result = await validate(payload, validators);
+
+  if (result.errors.length) {
+    ctx.status = 400;
+    ctx.body = {
+      errors: result.errors,
+    };
+
+    ctx.throw(400);
```

Comment:
We can use only one line here (If I remember correctly how it works.):
```javascript
ctx.assert(!result.errors, 400, result.errors);
```

Reply:
Seems, this solution doesn't work